### PR TITLE
docs: update documents for multiple level drilldown - `childGroupId`

### PR DESCRIPTION
**Pull request by tyn1998**
- [x] Chinese
- [x] English

modified:

- series.dataGroupId
- series.data.groupId
- series.data.childGroupId (Add)
- series.encode
- series.universalTransition

### CODE REVIEW

**Comment by tyn1998** [comment]
Hi @quillblue @pissang @Ovilia,

I have finished both en and zh docs for multiple levels drilldown(https://github.com/apache/echarts/pull/17611), please review the docs when you have time.

File: en/option/partial/group-id.md [hunk 1]
```diff
@@ -3,15 +3,59 @@
...
+If you are using the [dataset](~dataset) component to represent data, you are recommended to use `encode.itemGroupId` to specify the dimension that is to be encoded as the group ID.
+
+
+{{ target: partial-data-child-group-id }}
+
+#${prefix} childGroupId(string)
+
+The group ID of the child data of a data item. This option is introduced to make multiple levels drilldown and aggregation animation possilbe.
+
+~[700x300](${galleryViewPath}doc-example/bar-drilldown&edit=1&reset=1)
```
Comment (tyn1998) [inline en/option/partial/group-id.md:28]: This example will take effect after https://github.com/apache/echarts-examples/pull/55 is finished.

File: en/option/partial/group-id.md [hunk 2]
```diff
@@ -3,15 +3,59 @@
...
+
+Before `childGroupId` is introduced, developers actually can use `groupId` to make drilldown and aggregation animation already, but with the limit on the times that a continious drilldown or aggregation can happen, which is only one time.
+
+`childGroupId`, together with `groupId`, help to form "father-child" relationships between data items of different options, such as:
```
Comment (quillblue) [inline en/option/partial/group-id.md:32]: it would be better using `parent-child relationships`

File: en/option/partial/group-id.md [hunk 3]
```diff
@@ -3,15 +3,59 @@
...
 
 
 {{ target: partial-data-group-id }}
 
 #${prefix} groupId(string)
 
-The groupID of this data item. groupID will be used to classify the data.
+The group ID of a data item. When `universalTransition` is enabled, the data items from the old option and those from the new one, if sharing a same group ID, will then be matched and applied to a proper animation after `setOption` is called.
+
+If a data item is not specified with a `groupId`, Echarts will try to use `series.dataGroupId` as the group ID for the data item. If `series.dataGroupId` is not specified either, Echarts will fall back to using the data item's ID as its group ID.
```
Comment (quillblue) [inline en/option/partial/group-id.md:17]: Please be kind to use `Apache ECharts` as official name of ECharts (especially take care on letter C is in upper case)

**Review by quillblue — APPROVED** [review]
Looks good to me now.

File: en/option/partial/group-id.md (hunk 1 above)
Comment (plainheart) [reply en/option/partial/group-id.md:28]: It seems the new options don't have a version tip, such as `childGroupId` and `itemChildGroupId`. You can add it like this,

```
{{ use: partial-version(
    version = "5.5.0"
) }}
```

And the example link might need to be updated to the new `bar-multi-drilldown`.

Comment (tyn1998) [reply en/option/partial/group-id.md:28]: Got it, I will create another PR to address this :-)

**Pull request merged**